### DISCUSSION TITLE
501(c)(3) non-profit classification

### DISCUSSION THREAD

**Discussion by Brandon-Huu** [space-wizards/space-station-14#22164] [Questions] [answered]
Is this project registered as a non-profit organization?

If not, when will it be?

**Comment by deltanedas** [comment]
this is a github repository

**Comment by PJB3005** [comment, marked as answer]
It's currently not. We want to start using Open Source Collective (OSC) for financial things but haven't gotten around to it yet. Otherwise we have no plans to make any sort of legal entity like an organization for the project.
